dominos/src/main/java/me/fontys/semester4/dominos/DominosImport.java

```diff
@@ -19,10 +26,15 @@ public static void main(String[] args) {
     }
 
     private final StoreRepository storeRepository;
+    private final PCImporter pcImporter;
+    private final PCConverter pcConverter;
 
     @Autowired
-    public DominosImport(StoreRepository storeRepository) {
+    public DominosImport(StoreRepository storeRepository, PCConverter pcConverter, PCImporter pcImporter)
```

Comment:
Please see master branch for an example how we can easily invoke importers from the main class. Adding dependencies straight into the main file will become very cluttered (we'd have dozens of dependencies here eventually), and it becomes untestable; you can't invoke this at all from a unit test.

Reply:
There was no example when I made this pull request, I will take a look and rewrite.